lib/bibliothecary/path_info.rb

```diff
@@ -0,0 +1,28 @@
+module Bibliothecary
+  class PathInfo
```

Comment:
This means a "manifest-lockfile pair" or "a group of related files" right? I'm interpreting the name PathInfo to mean info about the `folder_path` that those related files are in... but you could have multiple PathInfo with the same folder_path right like package.json and Gemfile

Maybe something like `ManifestGroup` or `RelatedFilesInfo`? 

I originally was going to suggest a comment above `class PathInfo` briefly explaining what a PathInfo is, but I'm wondering if the right name could be a better solution than a comment.

Don't want to bikeshed names to death, but for someone reading this I'm not sure they'd know what this class semantically represents

Reply:
You're definitely right, I only really did PathInfo because the other was FileInfo. I really wanted to improve our searching for related manifest files so went this route. It was pointed out to me that technically, there could be three related i.e. ("package.json", "package-lock.json", "yarn.lock"). So I guess RelatedFilesInfo is a better name since we're distinguishing between manifest and lockfile?